logserver/src/test/java/alluxio/logserver/AlluxioLog4jSocketNodeTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.logserver;
+
+import static alluxio.logserver.AlluxioLog4jSocketNode.setAcceptList;
+
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.io.serialization.ValidatingObjectInputStream;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.apache.log4j.config.PropertySetterException;
+import org.apache.log4j.helpers.UtilLoggingLevel;
+import org.apache.log4j.lf5.Log4JLogRecord;
+import org.apache.log4j.lf5.LogLevel;
+import org.apache.log4j.lf5.LogLevelFormatException;
+import org.apache.log4j.lf5.util.AdapterLogRecord;
+import org.apache.log4j.lf5.viewer.LogTableColumn;
+import org.apache.log4j.lf5.viewer.LogTableColumnFormatException;
+import org.apache.log4j.pattern.LogEvent;
+import org.apache.log4j.spi.LocationInfo;
+import org.apache.log4j.spi.LoggingEvent;
+import org.apache.log4j.spi.ThrowableInformation;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.util.Hashtable;
+import java.util.List;
+
+public class AlluxioLog4jSocketNodeTest {
+
+  private byte[] mBuffer;
+  private ValidatingObjectInputStream mValidatingObjectInputStream;
```

Comment:
You don't need these member variables when they are only referenced in a single method.